yarn.lock

```diff
@@ -3077,16 +3077,16 @@ ansi-red@^0.1.1:
   dependencies:
     ansi-wrap "0.1.0"
 
-ansi-regex@^2.0.0, ansi-regex@^4.1.0, ansi-regex@^4.1.1:
-  version "4.1.1"
-  resolved "https://registry.yarnpkg.com/ansi-regex/-/ansi-regex-4.1.1.tgz#164daac87ab2d6f6db3a29875e2d1766582dabed"
-  integrity sha512-ILlv4k/3f6vfQ4OoP2AGvirOktlQ98ZEL1k9FaQjxa3L1abBgbuTDAdPOpvbGncC0BTVQrl+OM8xZGK6tWXt7g==
-
-ansi-regex@^3.0.1, ansi-regex@^6.0.1:
```

Comment:
I was expecting to see `jsonpath-plus` removed in this diff but it isn't

Reply:
```bash
jared:auth-js$ yarn why jsonpath-plus
yarn why v1.22.19
[1/4] 🤔  Why do we have the module "jsonpath-plus"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "jsonpath-plus@6.0.1"
info Reasons this module exists
   - "_project_#@okta#samples.express-embedded-sign-in-widget#@okta#okta-signin-widget#@okta#okta-auth-js" depends on it
   - Hoisted from "_project_#@okta#samples.express-embedded-sign-in-widget#@okta#okta-signin-widget#@okta#okta-auth-js#jsonpath-plus"
   - Hoisted from "_project_#@okta#samples.react-embedded-auth-with-sdk#@okta#okta-signin-widget#@okta#okta-auth-js#jsonpath-plus"
   - Hoisted from "_project_#@okta#samples.webpack-spa#@okta#okta-signin-widget#@okta#okta-auth-js#jsonpath-plus"
info Disk size without dependencies: "372KB"
info Disk size with unique dependencies: "372KB"
info Disk size with transitive dependencies: "372KB"
info Number of shared dependencies: 0
✨  Done in 0.56s.
```

Reply:
We include samples in the repo